doc/changeset.md

```diff
@@ -247,16 +247,16 @@ struct ArrayErase {
 }
 ~~~
 
-### ArrayClear (`0xb`)
+### Clear (`0xb`)
```

Comment:
Filed https://jira.mongodb.org/browse/REALMC-7409 to rename this in the backend as well, cc: @tkaye407